src/Entity/CommerceContentEntityBase.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\commerce\Entity;
+
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Language\LanguageInterface;
+
+/**
+ * Provides the base class for Commerce content entities.
+ */
+class CommerceContentEntityBase extends ContentEntityBase implements CommerceContentEntityInterface {
```

Comment:
How would stores need this? It doesn't have an architecture which has a concept of "child references"